content/articles/how-to-create-a-rest-api-using-spring-boot-maven-and-mysql/index.md

```diff
@@ -0,0 +1,810 @@
+
+### Key Takeaways
+
+* How to create and set up a spring boot application with a tomcat server.
+* MySQL Database Configuration in a Spring Boot project.
+* You will learn how to implement Object Relational Mapping using Hibernate.
+* Maven for dependency management.
+* Exception Handling in Java.
+* How to consume a rest API using postman.
+* How to document API using Swagger.
+
+### Prerequisites
+
+To fully understand this tutorial, you are required to have the following in place:
+* Good understanding of Java.
+* At least fundamental knowledge of the Spring framework and Spring Boot. However I will explain everything in detail, so if you are new to the Spring framework don’t be afraid I will be your friend on this journey.
+* [Java Development Kit(JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) installed to install.
+* [Postman](https://www.postman.com/downloads) installed or click on the link to download and install.
+* [Maven](https://maven.apache.org/download.cgi) installed or click on the link to download and install.
+* [IntelliJ](https://www.jetbrains.com/idea/download/) installed or click on the link to install. You can use any other IDEA of choice.
+
+### 1. Project Initialization and Setup
+
+There are different ways to set up a new spring boot application. One is using the [***spring initializer.***](https://start.spring.io).
+
+#### 1.1 Project Initialization and Setup Using IntelliJ
+
+In this tutorial I am going to show you how to initialize a spring boot project using Intellij. Intellij has an inbuilt feature to create a spring boot project that uses a Spring initializer which makes it easy to set up a project in no time at all. To get started Run IntelliJ IDEA and click create new project from File > New > Project.
+
+The next step is to select Spring initializer(highlighted with green. If you look at the section highlighted with red you will realize that we are connecting to the spring initializer on the web.
+
+![IntelliJ-Spring-initializer](/engineering-education/how-to-create-a-rest-api-using-spring-boot-maven-and-mysql/intellij-spring-initializer.png)
+Enter the Maven project properties of the project and click next
+* Give the project a name. I’m going to call mine employee_app
+* Choose the location where you want your project to reside. I’m going to keep mine on the desktop as you can see.
+* For Type of project, Choose  Maven.* The language for this project will be java, so were are going to select Java.
+* I will name the group “com.lekwacious” 
+* I will call artifact employee_app
```

Comment:
`I will name the group “com.lekwacious” ` -> `Let's name the group as “com.lekwacious” `

Always don't write from the first person's view, write from the 3rd person's view.